pyright/master/requirements-pinned.txt

```diff
@@ -558,8 +558,6 @@ tomli==2.0.1
 tomlkit==0.12.5
 toolz==0.12.1
 toposort==1.10
-torch==2.3.0
```

Comment:
any particular reason to remove these? I think omitting them is causing pyright errors in the build

Reply:
Bah sorry, it's because of this issue: https://github.com/dagster-io/dagster/issues/21800

I have to remove those deps in order to run "make pyright". I'll put it back